0.8.1: install target fails because missing man page

```console
+ cd xxHash-0.8.1
+ DESTDIR=/home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64
+ /usr/bin/cmake --install x86_64-redhat-linux-gnu
-- Install configuration: "RelWithDebInfo"
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/lib64/libxxhash.so.0.8.1
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/lib64/libxxhash.so.0
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/lib64/libxxhash.so
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/include/xxhash.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/include/xxh3.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/bin/xxhsum
-- Set runtime path of "/home/tkloczko/rpmbuild/BUILDROOT/xxhash-0.8.1-2.fc35.x86_64/usr/bin/xxhsum" to ""
CMake Error at x86_64-redhat-linux-gnu/cmake_install.cmake:119 (file):
  file INSTALL cannot find
  "/home/tkloczko/rpmbuild/BUILD/xxHash-0.8.1/cmake_unofficial/../xxhsum.1":
  No such file or directory.
```
cmake setup:
```console
[tkloczko@ss-desktop x86_64-redhat-linux-gnu]$ cmake -L
CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error: The source directory "/home/tkloczko/rpmbuild/BUILD/xxHash-0.8.1/x86_64-redhat-linux-gnu" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
-- Cache values
BUILD_SHARED_LIBS:BOOL=ON
CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
CMAKE_INSTALL_PREFIX:PATH=/usr
XXHASH_BUILD_XXHSUM:BOOL=ON
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -253,6 +253,8 @@ jobs:
         cd build
         cmake ..
         CFLAGS=-Werror make
+        mkdir test_install_dir
+        DESTDIR=test_install_dir cmake --install .
 
 
   # Linux, { ARM, ARM64, PPC64LE, PPC64, S390X }

--- a/cmake_unofficial/CMakeLists.txt
+++ b/cmake_unofficial/CMakeLists.txt
@@ -122,7 +122,7 @@ if(NOT XXHASH_BUNDLED_MODE)
     install(TARGETS xxhsum
       EXPORT xxHashTargets
       RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
-    install(FILES "${XXHASH_DIR}/xxhsum.1"
+    install(FILES "${XXHSUM_DIR}/xxhsum.1"
       DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
   endif(XXHASH_BUILD_XXHSUM)
 

